spec/internal/config/storage/development.yml

```diff
@@ -1,16 +1,14 @@
-minio: &minio
-  service: S3
-  access_key_id: <%= ENV['MINIO_ACCESS_KEY'] %>
-  secret_access_key: <%= ENV['MINIO_SECRET_KEY'] %>
-  region: <%= ENV['MINIO_REGION_NAME'] %>
-  endpoint: <%= ENV['MINIO_URL'] %>
-  force_path_style: true
+azurite: &azurite
+  service: AzureStorage
+  storage_account_name: <%= ENV['AZURE_STORAGE_ACCOUNT_NAME'] %>
+  storage_access_key: <%= ENV['AZURE_STORAGE_ACCOUNT_KEY'] %>
+  storage_blob_host: http://127.0.0.1:10000/<%= ENV['AZURE_STORAGE_ACCOUNT_NAME'] %>
```

Comment:
I can add that for development sake. But I don't think it will be required in prod

Reply:
Yeah, I think adding it would be a good idea, keep any hard-coded URLs out of the .env file, in case we start using a different service than the azurite docker container.